### PR TITLE
Separate stress tests out and run them on node 1

### PR DESCRIPTION
**Pull request by ararslan**
This ensures that they don't interfere with other tests. Suggested by @vtjnash in https://github.com/JuliaLang/julia/pull/28201#issuecomment-407520112. Let me know if any other tests belong here as well.

### CODE REVIEW

**Review by vtjnash — APPROVED** [review]
Perhaps `test_13559` and `test_22566` and the test for `jl_exit_on_sigint` also, if you're looking for more. I assume others will move or be added over time.

**Comment by iblis17** [comment]
`test_13559` and `test_22566` doesn't cause huge workload on pipe. So I hope the `file` testsuit won't freeze FreeBSD CI builds after this PR merged.

File: test/stress.jl [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        close(r)
+        @test success(p)
+        rm(fn)
+    end
+    test_13559()
+end
+
+# issue #22566
+# issue #24037 (disabling on FreeBSD)
+if !Sys.iswindows() && !(Sys.isbsd() && !Sys.isapple())
```
Comment (iblis17) [inline test/stress.jl]: Good to enable it on FreeBSD?

Comment (ararslan) [reply test/stress.jl]: I don't know, is it? 😄 If you reenable locally it can you reproducing the freezing it was causing before?

Comment (iblis17) [reply test/stress.jl]: > If you reenable locally it can you reproducing the freezing it was causing before?

I reenabled it.... seems fine now.

Comment (ararslan) [reply test/stress.jl]: Great! Thanks for checking, I'll reenable it here.

**Comment by iblis17** [comment]
I want to build at least 3 times to make sure no freezing again.

- [ ] https://freebsdci.julialang.org/#/builders/1/builds/11595
- [ ] TBD
- [ ] TBD

**Comment by ararslan** [comment]
There were test failures on Travis Linux (both 32- and 64-bit) related to Sockets not being able to connect, so I've restarted those builds to be safe. IIRC we don't run the Sockets tests at all on Circle.

**Comment by ararslan** [comment]
All good now. I'll go ahead and merge.

**Pull request merged**